### PR TITLE
Reload content menu item

### PR DESCRIPTION
**Pull request by Atlante45**
Added "View > Reload Content" menu item that clears all your caches and reloads all content except scripts.

### CODE REVIEW

**Comment by Atlante45** [comment]
Still need to do some testing on Release Mac/Windows, do not merge yet.

**Comment by hifi-gustavo** [comment]
Build console output for commit: ed296850e22fd26b419baf723e0e9e2e53d31c03 available at: https://gist.github.com/b2fb28d7b64b848ac674<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5276.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5276.dmg

**Comment by PhilipRosedale** [comment]
still testing, @Atlante45 ?

**Comment by Atlante45** [comment]
QProcess is not working as expected on Windows. It returns that it worked every time but the new process is never spawned.

I did some more research on other ways to do it and I found a way for us to clear the cache and all our resources without actually restarting the app, which I think would be a better user experience.
I need to make some more changes though, so that models don't try to update their joints while we are refreshing the assets otherwise we get a crash.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 49993695a5f7dc53486b37bfc05607f2e17cded1 available at: https://gist.github.com/b2fb28d7b64b848ac674<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5276.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5276.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: a3848f08d2c9d9da3639f90155df9759946a3925 available at: https://gist.github.com/b2fb28d7b64b848ac674<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5276.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5276.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: ec7e1df8c0ae2d0f720481bdf8b0fc1beacdcaac available at: https://gist.github.com/b2fb28d7b64b848ac674<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5276.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5276.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: c11864acf3ddf8faf0d8f831d90c5f0fc02edaf2 available at: https://gist.github.com/b2fb28d7b64b848ac674<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5276.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5276.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 7bf45308b66d7de41835f557f3a114b3c51fbafe available at: https://gist.github.com/b2fb28d7b64b848ac674<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5276.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5276.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: a32f54da2f04747776d6d7b101d1e89bc9c86d90 available at: https://gist.github.com/b2fb28d7b64b848ac674<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5276.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5276.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 8fb69ecc9620e3b7bef83abf72e51f277e2e9e16 available at: https://gist.github.com/b2fb28d7b64b848ac674<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5276.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5276.dmg

File: libraries/animation/src/AnimationCache.cpp [hunk 1]
```diff
@@ -39,8 +39,7 @@ QSharedPointer<Resource> AnimationCache::createResource(const QUrl& url, const Q
 }
 
 Animation::Animation(const QUrl& url) :
-    Resource(url),
-    _isValid(false) {
+    Resource(url) {
```
Comment (birarda) [inline libraries/animation/src/AnimationCache.cpp]: `{` on next line

File: libraries/entities/src/ModelEntityItem.cpp [hunk 2]
```diff
@@ -203,9 +203,8 @@ void ModelEntityItem::mapJoints(const QStringList& modelJointNames) {
         return;
     }
 
-    Animation* myAnimation = getAnimation(_animationURL);
-
-    if (!_jointMappingCompleted) {
```
Comment (birarda) [inline libraries/entities/src/ModelEntityItem.cpp]: do we still need to check `!_jointMappingCompleted`

Comment (Atlante45) [inline libraries/entities/src/ModelEntityItem.cpp]: It is already checked above.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 22f0dbdefb4efcb3f37f8f2ac775cbdee9132e67 available at: https://gist.github.com/b2fb28d7b64b848ac674<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5276.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5276.dmg

**Pull request merged**